doc/msvc.t2t

```diff
@@ -7,12 +7,17 @@ This is currently also how the binary QGIS packages are made (earlier versions u
 This section describes the setup required to allow Visual Studio to be used to
 build QGIS.
 
-===Visual C++ Community Edition===
+===Visual Studio 2015 Community Edition===
 
 The free (as in free beer) Community installer is available under:
 
 	http://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe
 
+Select "Custom" install and add the following packages:
+- "Common Tools for Visual C++ 2015" under "Visual C++"
```

Comment:
Required to avoid:
![visualC++2015tools](https://user-images.githubusercontent.com/34431661/81096064-b92cbe80-8eba-11ea-9b6d-be9a33203a0e.png)